packages/itwinui-react/src/core/ToggleSwitch/ToggleSwitch.tsx

```diff
@@ -26,6 +26,11 @@ export type ToggleSwitchProps = {
    * Icon inside the toggle switch. Shown only when toggle is checked.
    */
   icon?: JSX.Element;
+  /**
+   * Size of the toggle switch.
+   *  @default 'default'
+   */
+  size?: 'default' | 'small';
 } & Omit<React.InputHTMLAttributes<HTMLInputElement>, 'type'>;
```

Comment:
This should fix your test failure mentioned in https://github.com/iTwin/iTwinUI/pull/1092#discussion_r1123425633
```suggestion
} & Omit<React.InputHTMLAttributes<HTMLInputElement>, 'type' | 'size'>;
```

Reply:
Oh interesting! Thanks 😸